person_class.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @id = rand(100_000)
+    @rental = []
   end
 
   private
```

Comment:
- [ ] Kindly create a method `add_rental` in the `Person` class.  This method should take `Person`, `Date` and `Book` as parameters. The method should create a new Rental object. This would enable every person object to be able to create a new rental. Remember that `Book` and `Person` have a _**many to many**_ relationship through `Rental`. Therefore both `Book` and `Person` should be able to create a `Rental`.